### PR TITLE
Add :iS and :iS. commands to list sections (only iOS)

### PR DESCRIPTION
**Pull request by as0ler**
Creates the  following commands:

- :iS: List sections
- :iS*: List sections as r2 flags
- :iS.: List sections from the current module.

Only has support for iOS. The idea is to add ELF support for Android Native in the future.

### CODE REVIEW

File: src/agent/index.js [hunk 1]
```diff
@@ -2164,6 +2168,137 @@ function listMemoryRangesHere (args) {
...
+    .map(m => m.base)
+  return listSections(moduleAddr);
+}
+
+function listSectionsR2 (args) {
+  let i = 0;
+  return listSectionsJson(args)
+  .map(({ vmaddr, vmsize, name }) => {
+    return [`f section.${i++}.${name} ${vmsize} ${vmaddr}`].join(' ');
+  })
```
Comment (trufae) [inline src/agent/index.js:2184]: This needs filtering

File: src/agent/index.js [hunk 2]
```diff
@@ -2164,6 +2168,137 @@ function listMemoryRangesHere (args) {
...
+function listSections (args) {
+  return listSectionsJson(args)
+  .map(({ vmaddr, vmsize, name }) => {
+    return [vmaddr, vmsize, name].join(' ');
+  })
+  .join('\n');
+}
+
+function listSectionsJson (args) {
+  if (!ObjCAvailable && !SwiftAvailable) {
```
Comment (trufae) [inline src/agent/index.js:2197]: Seiftavailable is a function now

File: src/agent/index.js [hunk 3]
```diff
@@ -2164,6 +2168,137 @@ function listMemoryRangesHere (args) {
...
+    return 'Only iOS supported.';
+  }
+  let baseAddr = undefined;
```
Comment (trufae) [inline src/agent/index.js:2200]: Use terminary operator instead of assigning to undefined

File: src/io_frida.c [hunk 4]
```diff
@@ -427,6 +427,8 @@ static char *__system_continuation(RIO *io, RIODesc *fd, const char *command) {
 		":fd[*j] <address>           Inverse symbol resolution\n"
 		":i                          Show target information\n"
 		":iE[*] <lib>                Same as is, but only for the export global ones\n"
+		":iS[*]                      List sections\n"
+		":iS.                        List sections from the module at current offset\n"
```
Comment (trufae) [inline src/io_frida.c:431]: Is iSj also handled?

File: src/agent/index.js [hunk 5]
```diff
@@ -149,6 +149,10 @@ const commandHandlers = {
   isj: listSymbolsJson,
   'is*': listSymbolsR2,
 
+  iS: listSections,
+  'iS.': listSectionsHere,
+  'iS*': listSectionsR2,
+
```
Comment (trufae) [inline src/agent/index.js:155]: Add iSj and just call the Json function :)

File: src/io_frida.c (hunk 4 above)
Comment (as0ler) [reply src/io_frida.c:431]: iSj support added

File: src/agent/index.js (hunk 5 above)
Comment (as0ler) [reply src/agent/index.js:156]: there it is!

File: src/agent/index.js (hunk 1 above)
Comment (as0ler) [reply src/agent/index.js:2187]: filtered applied

**Pull request merged**